grakn-graql/src/main/java/ai/grakn/graql/internal/query/analytics/AbstractComputeQuery.java

```diff
@@ -73,24 +56,12 @@
 
     @Override
     public final T execute() {
-        GraknTx tx = tx().orElseThrow(GraqlQueryException::noTx);
```

Comment:
Just checking TODO now or later? and TODO what?

Reply:
Todo now I think. I will need to check with @duckofyork what the expected behaviour is if you run a compute query twice at once.